### PR TITLE
Testing non-headless for PDF links

### PR DESCRIPTION
**Pull request by benweese**
Wondering if the PDF's don't generate because it is headless.

### CODE REVIEW

File: src/tests/test_gcl_page.py [hunk 1]
```diff
@@ -36,7 +36,9 @@ def test_donate(browser):
 	page = GCLPOM(browser)
 	page.load()
 	page.test_donate()
-	assert browser.current_url == 'https://www.paypal.me/girlscodelincoln'
+	url = 'https://www.paypal.me/girlscodelincoln'
+	url2 = 'https://www.paypal.com/paypalme2/girlscodelincoln'
```
Comment (troynguyen8) [inline src/tests/test_gcl_page.py]: Why is this necessary?

File: src/tests/test_gcl_page.py [hunk 2]
```diff
@@ -249,7 +251,8 @@ def test_linked_in(browser):
 	page.load()
 	page.test_linked_in()
 	url = 'https://www.linkedin.com/company/girlscodelincoln'
-	assert browser.current_url == url
+	url2 = 'https%3A%2F%2Fwww.linkedin.com%2Fcompany%2Fgirlscodelincoln'
```
Comment (troynguyen8) [inline src/tests/test_gcl_page.py]: Is there a `encodeURI()` type function in python that you can leverage here instead of hard coding?

Comment (benweese) [reply src/tests/test_gcl_page.py]: I have updated the code to now have an encoder. 

This has the same issue as above because linkedIn likes to know where you came from to get to linkedIn it will send a giant link first so they can track where the link originated. This link has a hash that changed every time you click it. Inside the container it never fully goes to the linked in site and fails when I wait for the page title. Copy and pasting the link works fine to redirect you outside the container. 

In other words linkedIn and docker containers are stupid. Which is why I did it the way I did.

File: src/tests/test_gcl_page.py (hunk 1 above)
Comment (benweese) [reply src/tests/test_gcl_page.py]: URL is the correct code and when running outside the container it normally takes you to. However for some reason, unknown to me the 2nd url is hit most times inside the container. So in order to get it to pass inside and outside the container I had to ad the OR.
